helmfile/charts/opensearch/configurer/files/dashboards-resources/welcome.md

```diff
@@ -0,0 +1,25 @@
+# Welcome to Compliant Kubernetes!
+
+## Public docs
+
+In case you get lost, don't forget to check out the [public docs](https://elastisys.io/compliantkubernetes/). Here are the most common topics:
+
+- [Getting started](https://elastisys.io/compliantkubernetes/user-guide/prepare/)
+- [Adding extra workload admins](https://elastisys.io/compliantkubernetes/user-guide/delegation/#kubernetes-api)
+- [Troubleshooting](https://elastisys.io/compliantkubernetes/user-guide/troubleshooting/)
+- [FAQ](https://elastisys.io/compliantkubernetes/user-guide/faq/)
+
+## Compliant Kubernetes Version
+
+- Apps: **{{ .Values.dashboard.ck8sVersion }}** - [Release Notes](https://elastisys.io/compliantkubernetes/release-notes/)
+
+## Additional services:
+
+- [grafana.{{ .Values.baseDomain }}](grafana.{{ .Values.baseDomain }})
+- [opensearch.{{ .Values.baseDomain }}](opensearch.{{ .Values.baseDomain }})
+- [dex.{{ .Values.baseDomain }}](dex.{{ .Values.baseDomain }})
+- [harbor.{{ .Values.baseDomain }}](harbor.{{ .Values.baseDomain }})
+
+## Did you know?
+
+As a user of Compliant Kubernetes you can request to see your Cluster configuration (without secrets) by asking your administrator.
```

Comment:
It can be good to add a section `support`. With a link to jira and a support number etc. 
Other then that it looks good to me. :+1:

Reply:
Good idea